example/build.gradle

```diff
@@ -41,12 +41,16 @@ ext {
 dependencies {
     implementation project(path: ":core")
     implementation project(path: ":auth")
+    implementation project(path: ":push")
     implementation "com.android.support:appcompat-v7:$android_support_version"
     implementation "com.android.support:support-v4:$android_support_version"
     implementation "com.android.support:design:$android_support_version"
     implementation "com.android.support.constraint:constraint-layout:1.0.2"
+    implementation 'com.google.firebase:firebase-messaging:11.8.0'
```

Comment:
Should the firebase dependency be in the bom?

Reply:
Not in the sample app only on the library!

Reply:
I should remove this, push already have this as a dependency!